contracts/test/EIP1271Wallet.sol

```diff
@@ -1,111 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
We definitely don't want to delete this, need to be able to run tests against solidity and also want it as a point of reference

Reply:
multiple contracts with the same name would cause a name clash, need to think the project structure through

Reply:
For all the reference contracts we prefixed them with `Reference`, could just prefix with `Vyper` for the time being

Reply:
Will it still cause a name clash as vyper build folder is isolated from the solidity cache?